x-pack/plugins/lens/public/react_embeddable/renderer/lens_custom_renderer_component.tsx

```diff
@@ -112,7 +112,7 @@ export function LensRenderer({
   const panelProps: PanelProps = useMemo(() => {
     return {
       hideInspector: !showInspector,
-      hideHeader: showPanelChrome,
+      hideHeader: props.hidePanelTitles,
```

Comment:
`hideHeader` property disables the entire panel header; including titles, context menu, and hover actions. I don't think `hideHeader` is the best way to hide panel titles as its to broad and turns off other features.

The mechanism for hiding a panel title is [PublishesPanelTitle](https://github.com/elastic/kibana/blob/main/packages/presentation/presentation_publishing/interfaces/titles/publishes_panel_title.ts#L12) interface.

`hidePanelTitle` can come from the embeddable or the embeddable's parent. I would recommend putting `hidePanelTitle` on the parentApi since LensCustomRendererComponent has direct access to parentApi.

Instead of changin `hideHeader` in `panelProps`, add the following to the object created by [getParentApi](https://github.com/elastic/kibana/blob/main/x-pack/plugins/lens/public/react_embeddable/renderer/lens_custom_renderer_component.tsx#L133)
```
hidePanelTitle: new BehaviorSubject(props.hidePanelTitles)
```

Reply:
done here [`e8f8324` (#203355)](https://github.com/elastic/kibana/pull/203355/commits/e8f8324b2ea2738ccdb5abe4573ed096234010ce)